hooks/useInstallmentsData.ts

```diff
@@ -45,6 +45,16 @@ export default function useInstallmentsData({
         irmParameters,
         timestamp,
       ] as const;
+      // Pevents division by zero error in @exactly/lib/esm/installments/split function if there are no fixed pools
+      if (fixedPoolsUtilizations.length === 0) return;
+      // Check for already matured pools, @exactly/lib/esm/installments/split will throw an error if the pool is already matured
+      if (
```

Comment:
prefer not adding comments and naming the variable inside the `if` statement.
```
if (alreadyMatured) { ... }
```

Reply:
Hello @sebipap! code style fixed 😊